sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProtoTest.java

```diff
@@ -483,10 +484,10 @@ public void testNestedFromTableSchema() {
           .put("floatvalue", (double) 2.817)
           .put("boolvalue", true)
           .put("booleanvalue", true)
-          .put("timestampvalue", "43")
-          .put("timevalue", "00:52:07[.123]|[.123456] UTC")
-          .put("datetimevalue", "2019-08-16 00:52:07[.123]|[.123456] UTC")
-          .put("datevalue", "2019-08-16")
+          .put("timestampvalue", 43L)
+          .put("timevalue", "00:52:07.123456")
+          .put("datetimevalue", "2019-08-16T00:52:07.123456")
+          .put("datevalue", (int) LocalDate.parse("2019-08-16").toEpochDay())
           .put("numericvalue", "23.4")
```

Comment:
Could we test repeated byte string?  I always had some problems with it.

Reply:
@yirutang added extra test